### DISCUSSION TITLE
New to Dexcom G7; won't pair with Samsung Galaxy s10

### DISCUSSION THREAD

**Discussion by diaballer** [NightscoutFoundation/xDrip#3103] [Bug reports]
Hello. I have been using xdrip+ for the Dexcom G6 with gratitude for a few years. I am at the edge of my technical competence when I need to troubleshoot. I used to understand computers in the 90s and aughts! I tried looking for similar posts but could not find a solution that worked for me.

Can't pair Dexcom G7 with my Samsung Galaxy s10. It's my first time trying G7.

-Settings>Auto Configure does not open up a dialogue box about importing new settings. I get a camera view and reticle and nothing happens when I hold up the sensor QR code. 

-When I open Settings, I get a constant popup notification of "Please Allow Permission. Android 10 and above need background location for Bluetooth to work." BUT, to my knowledge, I have all relevant Bluetooth permissions enabled in my phone's Settings. Just double checked that. 

-Is there a way to Start Sensor manually like with a G6? 

Thank you for any help. Please let me know if I need to give more details. This is far, far from my day job (military historian).

![Screenshot_20230928-124440_Settings](https://github.com/NightscoutFoundation/xDrip/assets/146371630/1728dc0b-e15d-4cc5-8a7b-3ccd38ff2077)
![Uploading Screenshot_20230928-124245_xDrip+.jpg…]()

**Comment by Navid200** [comment]
G7 starts when you insert sensor.

The instructions are here: https://navid200.github.io/xDrip/docs/Dexcom/G7.html

**Reply by diaballer** [reply]
Hi Navid, thanks for your response. I already found that page, and as I wrote above, I get stuck at the 
Settings > Auto-Configure step. Hence, I stopped what I was doing and here I am, informing Nightscout and Jamorham, as that page instructs.

**Reply by Navid200** [reply]
Post a screenshot of what you see on screen when you tap on auto configure.

**Comment by Navid200** [comment]
This explains how to set up location access:  https://navid200.github.io/xDrip/docs/Location.html

You say you have trouble scanning the sensor.  
You are not supposed to.  You are supposed to scan the QR code in the guide.

**Comment by diaballer** [comment]
Thank you. I did turn on that permission for location.

Thanks also for clueing me in about which QR code to scan. Sadly I am still
stuck and can't scan.

When I hit "Auto Configure" within the xdrip+ Settings menu, my phone's
camera activates and I see a red line, and instructions to scan a QR code,
but no QR code ever seems to take.

**Reply by Navid200** [reply]
I may be able to help if I can see a screenshot of what you see when you tap on auto configure.
Without that, it is very hard to speculate what the problem may be.
